include/seqan/seq_io/sequence_file.h

```diff
@@ -387,30 +532,22 @@ writeRecord(FormattedFile<Fastq, Output, TSpec> & file,
  * @signature void writeRecords(fileOut, metas, seqs, quals);
  * @see SeqFileOut#writeRecord
  */
-
-template <typename TSpec, typename TIdStringSet, typename TSeqStringSet>
-inline void
-writeRecords(FormattedFile<Fastq, Output, TSpec> & file,
-             TIdStringSet const & meta,
-             TSeqStringSet const & seq)
+template <typename TFile, typename TIdStringSet, typename TSeqStringSet>
+inline void writeRecords(TFile & file, TIdStringSet const & meta, TSeqStringSet const & seq)
```

Comment:
If this was also the previous behaviour you can leave it like this, but maybe add a a TODO notice to the code that it should be changed.